tests/integration/cli/test_ssh.py

```diff
@@ -0,0 +1,45 @@
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import test Libs
+from tests.support.case import SSHCase
+
+
+class SSHCustomModuleTest(SSHCase):
```

Comment:
Maybe a different name for the class and comment from the tests/integration/cli/test_custom_module.py test as we are not testing a custom module?